client/src/pages/course/student/cross-check-review.tsx

```diff
@@ -107,30 +103,18 @@ function Page(props: CoursePageProps) {
     const [activeSolutionReview] = solutionReviews;
 
     form.setFieldValue('comment', activeSolutionReview.comment.slice(markdownLabel.length));
-    loadInitialCriteria(activeSolutionReview);
+    setScore(activeSolutionReview.score ?? 0);
```

Comment:
According to existing type annotations `activeSolutionReview.score` has type `number` and right part of expression is unnecessary